pymatbridge/pymatbridge.py

```diff
@@ -433,3 +500,64 @@ def _preamble_code(self):
 
     def _execute_flag(self):
         return '--eval'
+
+
+# ----------------------------------------------------------------------------
+# MATLAB METHOD
+# ----------------------------------------------------------------------------
+class Method(object):
```

Comment:
Yeah - let's go with that. Other than that - looks good to me. Let me give it another read tomorrow, before going ahead with this. Thanks a lot for taking this on! That split in the code-base has been bothering me for a long while.